dotcom-rendering/src/components/NewsletterCard.tsx

```diff
@@ -143,31 +202,48 @@ export const NewsletterCard = ({
 			css={groupItemStyle}
 			aria-label={newsletter.name}
 		>
-			<NewsletterDetail text={newsletter.frequency} />
-			<h3>{newsletter.name}</h3>
-			<p>{newsletter.description}</p>
-			<div css={buttonHolderStyle}>
-				<Button
-					{...getButtonInitialAriaAttribute(newsletter.name)}
-					priority="tertiary"
-					size="xsmall"
-					iconSide="left"
-					icon={
-						<>
-							<span className={ICON_PLUS_CLASS}>
-								<SvgPlus />
-							</span>
-							<span className={ICON_TICK_CLASS}>
-								<SvgCheckmark />
-							</span>
-						</>
-					}
-					cssOverrides={buttonStyle}
-					data-newsletter-id={newsletter.identityName}
-					data-role={BUTTON_ROLE}
-				>
-					Sign up
-				</Button>
+			{!!newsletter.illustrationUrl && (
+				<img
+					src={newsletter.illustrationUrl}
+					css={illustrationStyle}
+					alt=""
+					role="img"
+				/>
+			)}
```

Comment:
We generally don't use <img /> tags in DCR - We prefer <picture> elements with defined srcsets that support different images sizes depending on the browser. 

You can see a recent example of this here: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/TagFrontHeader.tsx#L252

There are lots of helper methods to make this quick to put together, but it's relevant that instead of using the pre-made images you've grabbed from the DOM on Frontend, that you use the original image URLs from CAPI, as DCR will take care of converting the URL format & adding parameters - you'll just need to define what widths of images you want at each breakpoint!

Reply:
Thanks - sounds good. One thing I'm not sure about is which the original URL is in the CAPI response - i think it would be the the url that appears in the "bodyHtml" field for the block, but can I check I've got that right?

EG on this article:
https://www.theguardian.com/australia-news/2022/oct/13/afternoon-update-newsletter-best-daily-news-email-guardian-australia-free-sign-up

the main block has this as "bodyHtml", so the right URL to provide to DCR would be "https://media.guim.co.uk/50b02d4e4a32def95d8d26cc852549e6bd83f037/0_0_1850_1111/1000.jpg" ?

```html
<figure class="element element-image" data-media-id="50b02d4e4a32def95d8d26cc852549e6bd83f037"> 
<img 
src="https://media.guim.co.uk/50b02d4e4a32def95d8d26cc852549e6bd83f037/0_0_1850_1111/1000.jpg" 
alt="Sign up for the Afternoon Update email newsletter, Guardian Australia’s free daily PM news briefing." 
width="1000" height="600" class="gu-image" /> 
<figcaption> 
<span class="element-image__caption">Sign up for the Afternoon Update email newsletter, Guardian Australia’s free daily PM news briefing.</span> <span class="element-image__credit">Illustration: Jack Walsh/The Guardian</span> 
</figcaption> 
</figure>
```